src/lib/address/yat.ts

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios'
+import { toChecksumAddress } from 'web3-utils'
+
+import { ResolveYat, ValidateYat } from './address'
+
+export const validateYat: ValidateYat = async ({ value }) =>
+  /^\p{Extended_Pictographic}{1,5}$/u.test(value)
+
+export const resolveYat: ResolveYat = async args => {
+  const { data } = await axios.get(
+    `https://octopus-app-mkjlj.ondigitalocean.app/emoji_id/${args.value}`,
```

Comment:
This API is used to avoid the CORS error. When we created basic proxy server to handle requests. And this is basic deployment on digital ocean without any YAT reference. 
Gomes: https://discord.com/channels/554694662431178782/954487574876020827/1006511894120640592
James: https://discord.com/channels/554694662431178782/954487574876020827/1007010278514372668

After domain is whitelisted on yat side. this can be reverted to yat endpoint. If there is env file, maybe we will extract it to env var and when YAT whitelists your domain, just edit env var to use yat endpoint

Reply:
I might have used a wrong wording here, by final I meant in the context of this PR - since it's most likely going to be landed to develop in this PR, we should comment it out so anyone reading it has context.

Speaking of context, thanks a lot for bringing context to the PR @saksijas !

Reply:
To confirm, create a comment explaining the api, why it looks like that and what is the usage. @gomesalexandre